dle/.env.example

```diff
@@ -6,6 +6,7 @@ MIGRATE=True
 LOAD=False
 # put Django into debug mode?
 DEBUG=True
+LOG_LEVEL=DEBUG
```

Comment:
Is this LOG_LEVEL same as the verbosity level passed to the parsing scripts? I found that debug level is not that useful (too many garbage logs)

Reply:
Yep you're totally right. I was doing some low level tracing and must have forgotten to change that in the `.env.example`. I'll update that